corehq/ex-submodules/soil/util.py

```diff
@@ -88,6 +92,20 @@ def get_download_context(download_id, message=None, require_result=False):
     }
 
 
+def process_email_request(download_id, email_address):
+    dropbox_url = absolute_reverse('dropbox_upload', args=(download_id,))
+    try:
+        allow_dropbox_sync = get_download_context(download_id).get('allow_dropbox_sync', False)
+    except TaskFailedError:
+        allow_dropbox_sync = False
+    download_url = "{}?get_file".format(absolute_reverse('retrieve_download', args=(download_id,)))
+    email_body = 'Your CommCare export is ready! Click on the link below to download your ' + \
```

Comment:
translate?

Reply:
same with the subject line below

Reply:
@xbryanc "translating" here means marking for translation. Every bit of text we show the user should be marked for translation, so that when a user who speaks French sees it, they see it in french. (In this particular case, there may be a bit more we need to do to make this user's language preference available at this point in the code, but we can handle that down the road—at least with it marked for translation our translators can translate it in the meantime.)

Marking for translation is explained here: https://docs.djangoproject.com/en/1.11/topics/i18n/translation/. In addition, the use of the `+` operator for string concatenation is strongly discouraged in python; use `'My favorite food is {}!'.format('bananas')`-style string formatting instead.